### DISCUSSION TITLE
GitHub student Developer Pack

### DISCUSSION THREAD

**Discussion by ManumiRansali** [community/community#130690] [GitHub Education]
### Hi everyone,

Who is eligible for GitHub Developer pack 
How can I apply for the Developer Pack

**Comment by ShenalSen** [comment]
Hi @ManumiRansali ,
1. If you are a student currently enrolled in a degree or dip in high school sec school or uni you are eligible for this offer.
2. And you should need your school issued verification document and email.
then go this site [GitHub Student Developer pack](https://github.com/edu/students) and complete the step. If you want more guidance about this you can connect with me on github.

Good luck!
